homeassistant/components/dynalite/bridge.py

```diff
@@ -1,118 +1,83 @@
 """Code to handle a Dynalite bridge."""
 
+import asyncio
+
 from dynalite_devices_lib import DynaliteDevices
-from dynalite_lib import CONF_ALL
+from dynalite_lib import CONF_ALL, CONF_HOST
```

Comment:
We should use the constant we have defined to access the config we have created. These constants are library specific and should just be used when passing an item to a library interface.

Reply:
fixed

Reply:
fixed